migrations/versions/3d55fce529de_add_details_to_item_itemcollection.py

```diff
@@ -0,0 +1,25 @@
+"""add details to item itemcollection
+
+Revision ID: 3d55fce529de
+Revises: 36f458047cfd
+Create Date: 2017-05-31 15:31:09.140120
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '3d55fce529de'
+down_revision = '36f458047cfd'
+
+from alembic import op
+import sqlalchemy as sa
+from coaster import sqlalchemy
```

Comment:
These two import invocations mean different things:
```python
from coaster import sqlalchemy
import coaster.sqlalchemy as sqlalchemy
```
For the first version to work, coaster's `__init__.py` must have an `import sqlalchemy`. This is obviously not a good idea. For `JsonDict`, always use `from coaster.sqlalchemy import JsonDict`